scripts/run-android.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -m # needed to access jobs
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+# generates android debug build when we pass the debug flag
+"${GIT_ROOT}"/scripts/build-android.sh
+
+# installs the resulting apk on running emulator or android device
+adb install ./result/app-debug.apk
```

Comment:
hmm probably we can also improve this bit by starting the android emulator if it is not already running 🤔

Reply:
maybe I'll do this in a follow up.

whenever I run `emulator -avd "$AVD_NAME"` I get this error : 

```
PANIC: Cannot find AVD system path. Please define ANDROID_SDK_ROOT
PANIC: Broken AVD system path. Check your ANDROID_SDK_ROOT value
```

There is nothing wrong with our `ANDROID_SDK_ROOT` value, perhaps we need to experiment with setting `ANDROID_AVD_HOME`

Reply:
We have a script for detecting this stuff written by @yakimant :
```
~/work/status-mobile develop ⇣
 > scripts/adb_devices_abis.sh    
adb_devices_abis.sh DEBUG: device_name=
adb_devices_abis.sh DEBUG: device_status=
adb_devices_abis.sh ERROR: no devices found. Check 'adb devices -l' output and share with Infra team if needed.

~/work/status-mobile develop ⇣
 > scripts/adb_devices_abis.sh
adb_devices_abis.sh DEBUG: device_name=1d60d861
adb_devices_abis.sh DEBUG: device_status=device
adb_devices_abis.sh DEBUG: device_product_abi=arm64-v8a
adb_devices_abis.sh DEBUG: resulting abis:
arm64-v8a
```